### ISSUE TITLE
Question about the depth prediction result

### ISSUE THREAD

**Issue by yangengt123** [tinghuiz/SfMLearner#42]
Hi tinghuiz, 
I tried your code recently. It worked cool! But after completing all the steps you stated on the website, I found a problem here. 

I believe I strictly followed your steps, and trained the model for 200K steps (the default value in your code). The evaluated depth accuracy I got was as follows: 
abs_rel,     sq_rel,        rms,        log_rms,     d1_all,         a1,         a2,         a3
 0.2331,     3.7935,     7.4710,     0.3076,     0.0000,     0.7006,     0.8807,     0.9453

I also tried to disable the pose net and fit  the ground truth pose to the model, and got the results shown below.
abs_rel,     sq_rel,        rms,       log_rms,     d1_all,         a1,         a2,         a3
 0.1889,     2.1794,     6.6112,     0.2708,     0.0000,     0.7413,     0.9053,     0.9597

It seems still worse than the result you post on the page. Is there any further fine-tuning should I make to reach the results rather than simply running the code directly?

Thanks!

**Comment by tinghuiz** [comment]
The final model I used was trained less than 200K steps (somewhere around ~160K I believe), as I noticed that the model started overfitting with longer training. As pointed out in another issue (https://github.com/tinghuiz/SfMLearner/issues/34), the weight decay is currently not applied correctly. Adding it could potentially alleviate the overfitting problem.
